circle.yml

```diff
@@ -1,54 +0,0 @@
-#
-# Copyright 2015-2018 The OpenZipkin Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
-# in compliance with the License. You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software distributed under the License
-# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions and limitations under
-# the License.
-#
-
-# NOTE: Travis tests Elasticsearch and MySQL using Docker, and Kafka manually. We don't redundantly test them here
-machine:
-  environment:
-    # Use java 9 here and 1.8 in travis (so that we test both)
-    JAVA_HOME: /usr/lib/jvm/zulu-9-amd64
-    # Quiet Maven invoker logs (Downloading... when running zipkin-server/src/it)
-    MAVEN_OPTS: -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-
-dependencies:
-  override:
-    - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x219BD9C9
-    - sudo apt-add-repository -y "deb http://repos.azulsystems.com/ubuntu stable main"
-    - sudo apt-get -y update
-    - sudo apt-get -y install zulu-9
-    - sudo apt-get install xsltproc
-    - ./circleci/go-offline.sh
-    - ./mvnw frontend:install-node-and-npm frontend:npm -pl zipkin-ui
-  cache_directories:
-    - "~/.npm"
-    - "zipkin-ui/node_modules"
-
-database:
-  override:
-    - mysql -u ubuntu -e 'SET GLOBAL innodb_file_format=Barracuda'
```

Comment:
I got rid of these as they did not seem needed given the note at the top about MySQL being tested on Travis.